### DISCUSSION TITLE
Support to upload to Google Drive.

### DISCUSSION THREAD

**Discussion by slamcoderpt** [filamentphp/filament#14077] [Ideas]
Is there a possibility to support uploads to a google drive folder?

**Comment by sinnbeck** [comment]
Yeah you can add a custom file system and use this package for the driver :)

https://github.com/spatie/laravel-google-cloud-storage

https://filamentphp.com/docs/3.x/forms/fields/file-upload#configuring-the-storage-disk-and-directory

**Reply by slamcoderpt** [reply]
Thanks for the help!
